src/app/page.tsx

```diff
@@ -67,14 +62,18 @@ export default function Home() {
 							people together from every corner of the globe.
 						</p>
 					</div>
-					<img
+					<Image
+						height={200}
+						width={200}
 						src="/assets/undraw1.svg"
 						alt=""
 						className="w-[200px] h-[200px] m-2"
 					/>
 				</div>
 				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
-					<img
+					<Image
```

Comment:
This can be a card just to avoid this unnecessary repetition

Reply:
Forget about the landing page. We will probably bring all that down and have a new one. We created the current one just for the sake of having a landing page.  No optimization is needed there for the moment

Reply:
Yes, breaking down is not about optimization, but keeping that code clean and maintanable, the frontend code can quekly become messy and hard to look into.

Reply:
No, I was not talking about optimization in terms of optimizing the site but in terms of the reusability of components. In simple terms, I did not want to create a component for that because the whole landing page will disappear in the future.